packages/ng/step-by-step.md

```diff
@@ -0,0 +1,284 @@
+## Create the component in /src
+
+```
+$ ng g module lol -a=src
+installing module
+  create src\app\lol\lol.module.ts
+$ ng g component lol -a=src --export=true
+installing component
+  create src\app\lol\lol.component.scss
+  create src\app\lol\lol.component.html
+  create src\app\lol\lol.component.spec.ts
+  create src\app\lol\lol.component.ts
+  update src\app\lol\lol.module.ts
+```
+
+> - `-a=src` tells ng-cli to use application src, so it will generate everything in `/src/app/...`. by default it will use the demo application so you have to specify if you want to create something in `/src`

```

Comment:
no cuz `npm start` also updates the generated doc, so id rather people use `npm start`

using `ng serve -a=demo` if you just cloned the repo wont work